app/views/pages/about.html.erb

```diff
@@ -40,7 +40,7 @@
 
 <div class="messageBody">
   <%= raw t('pages.about.message_text_three', {
-    linebreak_link: link_to(t('pages.about.linebreak'), 'https://linebreak.co', target: 'blank'),
+    linebreak_link: link_to(t('pages.about.linebreak'), 'http://linebreak.io', target: 'blank'),
```

Comment:
Looks like this is a different Linebreak! Maybe they shut down? In that case, I would update this translation key (`pages.about.linebreak`) so that you no longer have to pass in a link!

Reply:
Is it ok if I change the statement to the following:-
linebreak_link: t('pages.about.linebreak')
So now there is no link, just the "Linebreak" text.

Reply:
Exactly - we just want the Linebreak text!